docs/reference/logging/index.md

```diff
@@ -17,7 +17,7 @@ container's logging driver. The following options are supported:
 
 | `none`      | Disables any logging for the container. `docker logs` won't be available with this driver.                                    |
 |-------------|-------------------------------------------------------------------------------------------------------------------------------|
-| `json-file` | Default logging driver for Docker. Writes JSON messages to file.  No logging options are supported for this driver.           |
+| `json-file` | Default logging driver for Docker. Writes JSON messages to file.  two logging options are supported for this driver.          |
```

Comment:
Default Docker logging driver. Writes JSON messages to a file. This driver supports two options.